src/main/java/org/opensearch/security/configuration/CompatConfig.java

```diff
@@ -100,4 +105,15 @@ public boolean transportInterClusterAuthEnabled() {
             return true;
         }
     }
+
+    /**
+     * Returns true if passive transport auth is enabled
+     */
+    public boolean transportInterClusterPassiveAuthEnabled() {
+        final boolean interClusterAuthInitiallyPassive = transportPassiveAuthSetting.getDynamicSettingValue();
```

Comment:
How are we honoring the YML setting value that is set or are we ignoring it ?

Reply:
Setting it in TransportPassiveAuthSetting constructor